registration/templates/registration/form_checkbox_field.html

```diff
@@ -1,13 +1,14 @@
-<div class="row">
-    <div class="large-6 large-offset-1 columns end">
+<div class="page-content__inner__wrapper">
+    <div class="page-content__inner__left">
         {{ field.errors }}
+        <span class="wpcf7-form-control-wrap">
```

Comment:
+1 to removing the wordpress classes, although I know you already have it in mind